templates/sync-to-current.js

```diff
@@ -0,0 +1,96 @@
+// Merge all `package.extend.json` to the template and write to `package.json`.
+
+const { existsSync } = require('fs');
+const fs = require('fs/promises');
+const path = require('path');
+
+const templateJson = require('./package.json');
+
+const dependencyFields = [
+  'dependencies',
+  'devDependencies',
+  'peerDependencies',
+  'optionalDependencies',
+];
+
+const dependencyChanged = (json1, json2) => {
+  return !dependencyFields.every((field) => {
```

Comment:
How about using `.some()` to improve performance?

Reply:
hmm actually they are same